src/styles/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$dimension: 100px;
+$gap: 1px;
+$padding: 20px;
```

Comment:
usually if we create a variable it supposed to be reused later. So I think the naming should be more clear
For instance, for colors we use variables like `$primary` `$secondary` `$light-green` and so on.